### DISCUSSION TITLE
Request How To Guide "Reviewing disease mappings for Biocurators and clinicians"

### DISCUSSION THREAD

**Discussion by matentzn** [jamesaoverton/obook#414] [Tutorial requests]
I would like to onboard new people more rapidly to large scale review efforts where a tool like OAK generates a bunch of mapping candidates, and a reviewer determines wether the mapping is good or not. It would be awesome if one of our prolific curators could create a simple how to with 10 or less steps to get started, using the alignment of two clinical terminologies as an example (OMIM and Mondo is great). I am particularly interested in the thought processes and the resources involved (websites), and I am interested in the truth (google, Wikipedia use etc). 

@sabrinatoro @nicolevasilevsky would you be able to put something like this together?

**Comment by nicolevasilevsky** [comment]
when would you need this? I can work on it, as long as there is not a tight turnaround

**Comment by sabrinatoro** [comment]
I am happy to participate and put something together, but same question as
Nicole's: what is the timeline?

**Comment by matentzn** [comment]
No timeline, but @sabrinatoro we should build on your work here: https://github.com/mapping-commons/disease-mappings/issues/16

I would appreciate if you could provide a very crude 5 step process in this issue here that I can use to onboard a clinician for reviewing some mappings. It can be really really crude, just to get started, no more than 10 minutes work. Than we can iterate! We could even do this during the OBOOK hackathon, but the earlier I get something the better for me (secret project).

**Comment by matentzn** [comment]
I am really keen on doing something in this space, but maybe we all need to dig ourselves out of our wholes at the moment?
